dist/index.js

```diff
@@ -435,7 +443,7 @@ class DevServer {
     async startTscWatch() {
         this.log.notice('Starting tsc --watch');
         this.log.debug('Waiting for first successful tsc build...');
-        await this.spawnAndAwaitOutput('npm', ['run', 'watch:ts', '--', '--preserveWatchOutput'], this.rootDir, 'Watching for', { shell: true });
+        await this.spawnAndAwaitOutput('npm', ['run', 'watch:ts' /*, '--', '--preserveWatchOutput'*/], this.rootDir, /watching (files )?for/i, { shell: true });
```

Comment:
Why was the `preserveWatchOutput` flag necessary? Since I'm using `esbuild` (estrella), this causes errors, but I didn't see any downside to disabling it.

Maybe we should make the scripts, their flags and the test regexp/string configurable.

Reply:
The problem with watch:ts using tsc was that it would clear the screen with every build. Since we are "forwarding" stdout from watch:ts, this would clear the screen every time, thus we would not be able to see all other logs anymore.